release.yaml

```diff
@@ -61,7 +61,7 @@ rancher-vsphere-cpi:
   - 103.1.0+up1.7.0
 rancher-vsphere-csi:
   - 103.1.0+up3.1.2-rancher1
-  - 103.2.0+up3.1.2-rancher2
```

Comment:
Please, keep only the `rancher-vsphere-csi` version that you're currently working on and the one that should be released in the upcoming _ Rancher v2.8.3_ Release.